.github/workflows/go.yml

```diff
@@ -14,12 +14,12 @@ jobs:
       max-parallel: 1
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        go_version: ['1.15', '1.16']
+        go_version: ['1.17', '1.18']
```

Comment:
If you're gearing up for a release, it may be worth updating the README as well. It still suggests a dependency on `1.14`. I don't think we can guarantee that if our CI is on `1.17+`.

Reply:
we can update the go version to 1.15+ in README after the new release. since 1.15 and 1.16 have been stopped support, I don't think we need to test them in the CI. We only need to test the latest two major versions according to the go release policy https://go.dev/doc/devel/release#policy